src/main/java/CompanyStats.java

```diff
@@ -0,0 +1,52 @@
+public class CompanyStats {
+    public double getAvgSalary(Employee[] employees) {
+        double totalSalary = 0;
+        int employeesNo = 0;
+
+        for (Employee employee : employees) {
+            totalSalary += employee.getSalary();
+            employeesNo++;
+        }
+
+        try {
+            return totalSalary / employeesNo;
+        } catch (ArithmeticException e) {
+            return 0;
+        }
```

Comment:
Jeżeli dzielenie wykonujesz na doublach (albo chociaż jednym double i zajdzie promocja typu), to nigdy nie będzie rzucony ten wyjątek. Ot taka ciekawostka, że tylko intów nie można dzielić przez 0